cli/config/config.go

```diff
@@ -16,8 +16,8 @@ import (
 const (
 	// ConfigFileName is the name of config file
 	ConfigFileName = "config.json"
-	configFileDir  = ".balena"
-	oldConfigfile  = ".balenacfg"
+	configFileDir  = ".balena-engine"
+	oldConfigfile  = ".balena-engine-cfg"
```

Comment:
What is `oldConfigfiile`? It sounds like renaming this will cause the new cli to just ignore existing configs, is that right?